e2e/testsuite/testconfig.go

```diff
@@ -413,18 +404,17 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
 				Version:    cc.Tag,
 			},
 		},
-		Bin:                    cc.Binary,
-		Bech32Prefix:           "cosmos",
-		CoinType:               fmt.Sprint(sdk.GetConfig().GetCoinType()),
-		Denom:                  denom,
-		EncodingConfig:         SDKEncodingConfig(),
-		GasPrices:              fmt.Sprintf("0.00%s", denom),
-		GasAdjustment:          1.3,
-		TrustingPeriod:         "508h",
-		NoHostMount:            false,
-		ModifyGenesis:          getGenesisModificationFunction(cc),
-		ConfigFileOverrides:    configFileOverrides,
-		UsingNewGenesisCommand: useNewGenesisCommand,
```

Comment:
how is backwards compatibility being handled now?

Reply:
had the same thoughts wrt backwards compat, but fingers crossed the internal lookup in interchaintest handles this across the board